diagrams/onprem/ci.py

```diff
@@ -35,6 +39,7 @@ class Zuulci(_Ci):
 # Aliases
 
 CircleCI = Circleci
+ConcourseCI = Concourseci
```

Comment:
Please add a node via `autogen.sh`, you should add an alias in `config.py`.

Please read the contribution guideline: https://github.com/mingrammer/diagrams/blob/master/CONTRIBUTING.md

Reply:
Okay, will do.